src/runtime_src/doc/toc/security.rst

```diff
@@ -187,10 +187,13 @@ partition is not fully AXI-compliant or deadlocks/stalls/hangs during operation.
 partition fails, AXI Firewall *trips* -- it starts completing AXI transactions on behalf of the slave so the
 master and the specific AXI bus is not impacted -- to protect the Shell. The AXI Firewall starts completing
 all transactions on behalf of misbehaving slave while also notifying the mgmt driver about the trip. The
-xclmgmt driver then starts taking recovery action. xclmgmt posts a message to xocl using MailBox to inform
-the peer about FireWall trip. xocl can suggest a reset by sending a reset command to xclmgmt on MailBox. Note
-that even if no reset is performed the AXI Protocol Firewall will continue to protect the host PCIe bus. DFX
-partition will be unavailable till device is reset. **A reboot of host is not required to reset the device.**
+xclmgmt driver then starts taking recovery action. xclmgmt posts a XCL_MAILBOX_REQ_FIREWALL message to xocl using
+MailBox to inform the peer about FireWall trip. xocl can suggest a reset by sending a XCL_MAILBOX_REQ_HOT_RESET message
+to xclmgmt on MailBox. Note that even if no reset is performed the AXI Protocol Firewall will continue to protect the host PCIe bus.
+DFX partition will be unavailable till device is reset. **A reboot of host is not required to reset the device.**
+
+On board that has multiple FPGA devices on one board and supports card level reset like u30, message exchanges on mailbox are exactly
```

Comment:
On board that has multiple FPGA devices on one board and supports card level reset like u30 --> Alveo boards with multiple FPGA devices on the same board like U30 support card level. 

message exchanges on mailbox are exactly the same to single FPGA board --> Mailbox usage by each device on the card is similar to that of single device cards, however firewall trip in device will trigger reset all devices on the card.

Reply:
changed